changes/4898.fixed

```diff
@@ -0,0 +1,2 @@
+Corrected `prefetch_related`/`select_related` calls to improve performance of `/api/ipam/prefixes/` and `/api/ipam/vrfs/` REST API endpoints.
+Corrected `select_related` calls to improve performance of `/ipam/namespaces/`, `/ipam/route-targets/`, and `/ipam/vrfs/` UI list views.
```

Comment:
Do you think this should just be "Optimized database queries to improve performance of ..." to make this easier to understand for non-Django developers reading the release notes?

Reply:
Yeah, probably so.

```suggestion
Optimized database queries to improve performance of `/api/ipam/prefixes/` and `/api/ipam/vrfs/` REST API endpoints.
Optimized database queries to improve performance of `/ipam/namespaces/`, `/ipam/route-targets/`, and `/ipam/vrfs/` UI list views.
```